### PR TITLE
Design improvements(?)

### PR DESCRIPTION
**Pull request by bilgeonceken**
comparison

### CODE REVIEW

**Comment by duck2** [comment]
On a first sight, not a deep inspection:

- The red is too oibs 
- I would resist dropping the depressive and succinct style(capital letters?!)

However the form default value thing and the current course wrapper looks nice.

**Comment by duck2** [comment]
perhaps this needs a CSS reset, wtf sans serif font is this destroys the overall "my sun is fading" attitude

**Comment by bilgeonceken** [comment]
Current theme is scaring potential users away. So i believe some kind of visual change is necessary including capital letters and a new font to look more user friendly. I also want to add a table header but other elements show strange behavior when i try to do that in a "normal" way or "not so hacky" way. It can be done in a "kinda hacky" way though, unless we decide to build the css structure again from scratch.

**Comment by duck2** [comment]
I won't deny this PR exposes some design deficiencies, but introducing an Ubuntu/OIBS red, rounding the edges and capitalizing first letters don't look necessarily userfriendly to me.

Issues I notice are:
- in Firefox, text, form, buttons and boxes are not really aligned
- a CSS reset is desperately needed because of the goofy button styling some OS/browsers use
- a border would be nice for "current courses"
- course colors are too much of one saturation- violet/blue/green

also a grid and headers for the table could be good. a SVG grid maybe?

**Comment by bilgeonceken** [comment]
newstyle

**Comment by duck2** [comment]
I don't think there is a point in selecting a theme now, let's first do the layout and then the colors.
More issues are:

- basbursen abomination keeps next/prev buttons aligned with the page on mobile, we don't
- flex wrapping can be unreliable, perhaps migrate to [PureCSS](purecss.io)? I think that will result in an actually smaller style.css. it is a mess now.

File: index.html [hunk 1]
```diff
@@ -1,84 +1,111 @@
...
-<link rel="icon" href="favicon.ico" type="image/x-icon">
-<link rel="stylesheet" href="style.css">
-<link rel="stylesheet" href="awesomplete.css">
-<div class="header">
-<a id="index">tetick.xyz</a>
-<a id="save">save</a>
-<a id="about-link">about</a>
-<a id="wrong-link">wrong!</a>
-</div>
+<html lang="en">
```
Comment (duck2) [inline index.html]: there is no point serving those, the browser injects them. see google's [style guide](https://google.github.io/styleguide/htmlcssguide.xml).

File: index.html [hunk 2]
```diff
@@ -1,84 +1,111 @@
...
-	arrow keys also work.
-</div>
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-<script src="awesomplete.js"></script>
-<script src="data.js"></script>
-<script src="main.js"></script>
+  <!-- FONT
```
Comment (duck2) [inline index.html]: let's keep the HTML comment-free please. it should be self explanatory.

File: index.html [hunk 3]
```diff
@@ -1,84 +1,111 @@
...
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```
Comment (duck2) [inline index.html]: let's keep the HTML webfont-free please. system fonts are enough for most things, they download 0kb and they don't jitter or show blank text while the actual font is being downloaded.

File: index.html [hunk 4]
```diff
@@ -1,84 +1,111 @@
...
+                    <button class="stuffbutton" id="nextb">Next</button>
+                </div>
+        		<div id="fdate" style="text-align: center; color: #aaa; font-size: 15px;">course data is from </div>
+                <div id="state">blank</div>
+            </div>
+        </div>
+        <div class="stuff">
+            <div class="options">
+                <div>Department: <input style="position:/*
+.eatd{position: relative; height: 100%; cursor: pointer;}*/relative; left:44px;" id="dept" class="" placeholder="EE, MINE etc."></div>
```
Comment (duck2) [inline index.html]: no px-oriented and/or long inline styling please. most CSS frameworks have stuff for aligning forms.

File: style.css [hunk 5]
```diff
@@ -1,93 +1,98 @@
-html, body {
-	margin: 0px;
-	padding: 0px;
-	height: 100%;
-	font-family: sans-serif;
-	font-size: 10pt;
+header{
```
Comment (duck2) [inline style.css:1]: about the header- it is designed be small and fit on mobile screens without requiring flex wrap. this layout won't fit on mobile screens and will look weird. try resizing the browser below 400px width.

File: style.css [hunk 6]
```diff
@@ -102,77 +107,171 @@ td{min-width: 8em; border: 1px solid black; height: 25em;}
...
+.toggle{
+    width: 6em;
+    height: 2em;
+    line-height: 1em;
+    padding: 0;
+    position: relative;
+    bottom: 5px;
+}
+
+/* COPY PASTE FROM LIBDUCK. PROPABLY RELATED WITH SOME THINGS*/
```
Comment (duck2) [inline style.css:288]: I know this is not an actual comment but let's keep the CSS comment free too. anlayan anlasın anlamayan karpuz yesin aq

**Comment by bilgeonceken** [comment]
I should not have pushed to this branch. Not only i hated it but i also destroyed the consistent flexbox structure after i added(copy pasted) the styling for course list to make a workaround for something that does not need a workaround kind of solution.  Working at night time is bad. I'll try to create something else if i find time. border-collapse is buggy by the way. Borders disappear sometimes.

**Comment by bilgeonceken** [comment]
PureCSS +1

**Comment by bilgeonceken** [comment]
out

**Comment by bilgeonceken** [comment]
Nevermind the commits. I still hate it

**Comment by duck2** [comment]
colors intensify hate but hard to change when they are coupled with layout xd

I'll push a clean slate purecss layout if I can find enough time and motivation.

also no `px` please it makes the layout brittle like biscuit

**Comment by aybberk** [comment]
bence bilgenin dizaynları so gay :)

**Pull request closed without merging**